src/components/Tabs/Tabs.stories.js

```diff
@@ -0,0 +1,236 @@
+import React from 'react';
+import ClassNames from 'classnames';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, select, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { defaultTemplate } from '../../../storybook/decorators/storyTemplates';
+import {
+  TabContainer,
+  Nav,
+  NavItem,
+  NavDropdown,
+  MenuItem,
+  TabPane,
+  TabContent,
+  Col,
+  Row,
+} from '../../index';
+
+const stories = storiesOf('Tabs', module);
+const description = (
+  <p>
+    This component is based on React Bootstrap Tabs component. See{' '}
+    <a href="https://react-bootstrap.github.io/components.html#tabs">
+      React Bootstrap Docs
+    </a>{' '}
+    for complete Tabs component documentation.
+  </p>
+);
+stories.addDecorator(withKnobs);
+stories.addDecorator(
+  defaultTemplate({
+    title: 'Tabs',
+    documentationLink:
+      'http://www.patternfly.org/pattern-library/widgets/#tabs',
+    description: description,
+  }),
+);
+
+stories.addWithInfo('Basic Tabs', () => {
+  const navStyle = select(
+    'Navigation Style',
+    ['tabs', 'patternfly style'],
+    'tabs',
+  );
+  const bsClass = ClassNames('nav nav-tabs', {
+    'nav-tabs-pf': navStyle !== 'tabs',
+    'nav-justified': boolean('Justified', false),
+  });
+  const t1Disabled = boolean('Tab 1 Disabled', false);
+  const t1Title = (
+    <div
+      dangerouslySetInnerHTML={{
+        __html: text('Tab 1 Tittle', 'Tab 1'),
+      }}
+    />
+  );
+  const t2Disabled = boolean('Tab 2 Disabled', false);
+  const t2Title = (
+    <div
+      dangerouslySetInnerHTML={{
+        __html: text('Tab 2 Tittle', 'Tab 2'),
+      }}
+    />
+  );
+  const t3Disabled = boolean('Tab 3 Disabled', false);
+  const t3Title = (
+    <div
+      dangerouslySetInnerHTML={{
+        __html: text('Tab 3 Tittle', 'Tab 3'),
+      }}
+    />
+  );
+
+  return (
+    <TabContainer id="basic-tabs" defaultActiveKey={1}>
+      <Row className="clearfix">
+        <Col sm={12}>
+          <Nav bsClass={bsClass} onSelect={action('selected')}>
+            <NavItem eventKey={1} disabled={t1Disabled}>
+              {t1Title}
+            </NavItem>
+            <NavItem eventKey={2} disabled={t2Disabled}>
+              {t2Title}
+            </NavItem>
+            <NavItem eventKey={3} disabled={t3Disabled}>
+              {t3Title}
+            </NavItem>
+          </Nav>
+        </Col>
+        <Col sm={12}>
+          <TabContent animation>
+            <TabPane eventKey={1}>Tab 1 content</TabPane>
+            <TabPane eventKey={2}>Tab 2 content</TabPane>
+            <TabPane eventKey={3}>Tab 3 content</TabPane>
+          </TabContent>
+        </Col>
+      </Row>
+    </TabContainer>
+  );
+});
+
+stories.addWithInfo(
+  'Tabs with Dropdown',
+  'Note: This variation is not keyboard accessible and is not recommended for use where accessibility is a priority.',
```

Comment:
I think it's fine to include this note under Show Info, but my concern is that it's a little buried and can be easily missed. 

Can we also add this note to the description for this storybook variation? So that it displays like this: 
![image](https://user-images.githubusercontent.com/21063328/34164382-660024e6-e4a7-11e7-8946-7642b128eb6c.png)

And can we add this description to just the Tabs with Dropdown example and not the other Tabs examples?